react/features/subtitles/middleware.ts

```diff
@@ -140,12 +143,13 @@ function _endpointMessageReceived({ dispatch, getState }: IStore, next: Function
                 txt.unstable = text;
             }
 
-            APP.API.notifyTranscriptionChunkReceived({
-                messageID: transcriptMessageID,
-                language: json.language,
+            notifyTranscriptionChunkReceived(
+                transcriptMessageID,
+                json.language,
                 participant,
-                ...txt
-            });
+                txt,
+                store
+            );
 
             // Dump transcript in a <transcript> element for debugging purposes.
             if (!json.is_interim && dumpTranscript) {
```

Comment:
Please check if we are on web here, since we are using DOM APIs that won't work on mobile.